debian/libmiroil1.symbols

```diff
@@ -21,3 +21,6 @@ libmiroil.so.1 libmiroil1 #MINVER#
  (c++)"miroil::PersistDisplayConfig::~PersistDisplayConfig()@MIROIL_1.0" 2.2.0
  (c++)"typeinfo for miroil::DisplayConfigurationPolicy@MIROIL_1.0" 2.2.0
  (c++)"vtable for miroil::DisplayConfigurationPolicy@MIROIL_1.0" 2.2.0
+ MIROIL_2.2@MIROIL_2.2 2.2.0
```

Comment:
You should't need this

Reply:
Okey, removed the line manually, and "make check-miroil-symbols" doesn't recreate it when I ran it.

It appear after first time I ran the package build.